README.md

```diff
@@ -64,6 +64,9 @@ keys:
 * `scope` - If a name is something like `@org/module` then the `scope`
   field will be set to `@org`.  If it doesn't have a scoped name, then
   scope is `null`.
+* `scopeWithoutName` - If a name is something like `@org/module` then the `scopeWithoutName`
```

Comment:
I think this is named wrong?  It's `nameWithoutScope` in the code, but `scopeWithoutName` here.